### DISCUSSION TITLE
TMP website

### DISCUSSION THREAD

**Discussion by jo99jo** [onemen/TabMixPlus#345] [General]
It looks like the old https://tabmixplus.org is defunct.  Is Github the current support for TMP?  

And is the version here only for the latest FF browsers?

**Comment by onemen** [comment]
Yes

Tab mix work on all versions from Firefox ESR 115 up to the latest version

**Reply by jo99jo** [reply]
Doesn't appear to work.  I drag the XPI file to FF 131.0.3 x64 on Win 10.  I get error "This add-on could not be installed because it appears to be corrupt."

And how would I import my TMP tabs from the old browser?

**Reply by onemen** [reply]
Read our docs
https://onemen.github.io/tabmixplus-docs/other/installation/

**Reply by jo99jo** [reply]
It installed in Waterfox but this is far too complicated!  I don't want to be a developer.  I just want an extension that works.  I will check in at some future time to see if a simpler process has been developed.

**Reply by pyrates999** [reply]
Waterfox G3 I believe has made the adjustments that the steps require so that you can install the extension.

**Reply by jo99jo** [reply]
Even after I installed it, I had already installed the imported tabs as bookmarks but couldn't see how to create tabs or where the options were.

Maybe I'm just thick but I've been working with computers and technology for 40+ years.

**Reply by pyrates999** [reply]
Tools -> tab mix plus options
